en/intro_to_command_line/README.md

```diff
@@ -185,14 +185,30 @@ Music
 
 Now, let's go to our Desktop directory:
 
-<!--sec data-title="Change current directory: OS X and Linux" data-id="OSX_Linux_move_to" data-collapse=true ces-->
+<!--sec data-title="Change directory: OS X" data-id="python_OSX"
+data-collapse=true ces-->
 
 {% filename %}command-line{% endfilename %}
 ```
 $ cd Desktop
 ```
 <!--endsec-->
 
+<!--sec data-title="Change directory: Linux" data-id="python_linux"
+data-collapse=true ces-->
+
+{% filename %}command-line{% endfilename %}
+```
+$ cd Desktop
+```
+
+(The directory name "Desktop" might be translated
+to the language of your Linux account.
+If that's the case, you'll need to replace `Desktop`
+with the translated name, e.g. `Schreibtisch` for German.)
```

Comment:
The Latin abbreviation "e.g." is problematic for many English speakers. They don't really know what it means, and it's often confused with "i.e.". Also, the punctuation here is incorrect -- it has to be punctuated as though it was "for example". So if you want to use "e.g." (which I recommend against), this line needs to be:

with the translated name; e.g., `Schreibtisch` for German.)

Also I am wondering why this whole paragraph is in parentheses? I think it would be better to take out the parens (didn't we have an issue/PR about that a month or so ago?) and start the paragraph with "Note that the directory...".

Reply:
> The Latin abbreviation "e.g." is problematic for many English speakers. They don't really know what it means, and it's often confused with "i.e.". Also, the punctuation here is incorrect -- it has to be punctuated as though it was "for example". So if you want to use "e.g." (which I recommend against), this line needs to be:
> 
> with the translated name; e.g., `Schreibtisch` for German.)

I've expanded the abbreviation to "for example". Is the current punctuation correct for that, or should it be "...<strong>;</strong> for example<strong>,</strong> ..." instead (or something yet else)?

Reply:
> Also I am wondering why this whole paragraph is in parentheses?

The [original remark](https://github.com/DjangoGirls/tutorial/pull/1465/files#diff-0ce35d3afd321211a71112aa5f36d20aL605) was also in a parenthesis. (Even one of the dreaded full-sentence-within-another-full-sentence kind.) I adopted that here, because the remark is only relevant for part of the readership (those reading the English translation despite having their Linux account language set to something else), but now that you point it out I agree that "Note that ..." without parenthesis would be more appropriate. I'll change it accordingly.

Reply:
> I agree that "Note that ..." without parenthesis would be more appropriate. I'll change it accordingly.

Done: 15723bfb0d23dd8897eba852733904fb48b78093

Reply:
...; for example, ... 
is indeed the correct punctuation.

Reply:
fixed in ee1169a131f926439115bc98cdc3b9915a9bfecc